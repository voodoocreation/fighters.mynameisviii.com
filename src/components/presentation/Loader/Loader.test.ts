import ComponentTester from "../../../utilities/ComponentTester";

import Loader from "./Loader";

const component = new ComponentTester(Loader).withDefaultProps({
  className: "TestLoader"
});

describe("[presentation] <Loader />", () => {
  it("renders correctly", () => {
    const { actual } = component.render();

    expect(actual).toMatchSnapshot();
  });
});
