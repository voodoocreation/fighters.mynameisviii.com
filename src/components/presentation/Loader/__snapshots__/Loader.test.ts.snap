// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[presentation] <Loader /> renders correctly 1`] = `
<span
  class="Loader TestLoader"
>
  <div
    class="Loader--animation"
  >
    <span />
    <span />
    <span />
  </div>
</span>
`;
